Makefile

```diff
@@ -48,9 +48,12 @@ endef
 
 define in_docker
 	docker run \
+	--name prestashop-flashlight-${PS_VERSION} \
 	--workdir /var/www/html/modules/${MODULE_NAME} \
 	--volume $(shell pwd):/var/www/html/modules/${MODULE_NAME}:rw \
-	--entrypoint $1 ${TESTING_IMAGE} $2
+	--entrypoint $1 ${TESTING_IMAGE} $2 \
+	
+	docker rm prestashop-flashlight-${PS_VERSION}
```

Comment:
If you want to delete the container at the end of its run, it would be better to directly use the `--rm` flag in your docker run command.